ctapipe_io_nectarcam/__init__.py

```diff
@@ -80,6 +80,38 @@ def __init__(self, **kwargs):
 
         self.log.info("Read {} input files".format(self.multi_file.num_inputs()))
 
+    def subarray(self):
+        """
+	Obtain the subarray from the EventSource
+        Returns
+        -------
+        ctapipe.instrument.SubarrayDecription
+        """
+
+	tel_id = 1
+
+        # optics info from standard optics.fits.gz file
+        optics = OpticsDescription.from_name("MST")
+
+        # camera info from NectarCam-[geometry_version].camgeom.fits.gz file
+        geometry_version = 3
+        camera = CameraGeometry.from_name("NectarCam", geometry_version)
+
+        tel_descr = TelescopeDescription(
+            name='MST', tel_type='MST', optics=optics, camera=camera
+        )
+
+	tels = {tel_id: tel_descr}
+
+        # LSTs telescope position taken from MC from the moment
+        tel_pos = {tel_id: [0., 0., 9.] * u.m}
```

Comment:
Probably should be an MST position - you could always just set it to 0,0,0 for now, and set the name of the array as "Adlershof" or something, since so far you only have data from there.